_posts/2022-05-31-scala-on-app-service.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Scala on App Service"
+toc: true
+toc_sticky: true
+author_name: Denver Brittain
+excerpt: "Run a Scala App using Java SE on App Service"
+category: java
+---
+
+## Prerequisites
+
+This sample is based off of the [Play Framework Hello World Tutorial](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-hello-world-tutorial). 
+
+To follow the steps in this tutorial, you will need the correct version of Java, sbt, and (optionally) Maven or the Azure CLI.
+
+* Java 11
+* sbt 1.3.4 or higher. Note: if you downloaded this project as a zip file from <https://developer.lightbend.com>, the file includes an sbt distribution for your convenience.
+* Maven (Optional)
+* Azure CLI (Optional)
+
+To check your Java version, enter the following in a command window:
+
+```bash
+java -version
+```
+
+To check your sbt version, enter the following in a command window:
+
+```bash
+sbt sbtVersion
+```
+
+If you do not have the required versions, follow these links to obtain them:
+
+* [Java SE](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [sbt](http://www.scala-sbt.org/download.html)
+* [maven](https://maven.apache.org/install.html) (Optional)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) (Optional)

```

Comment:
you can move these hyperlinks into the bullets on lines 16-19, cuts down on length